src/auth/views/auth/join-almost/join-almost.html

```diff
@@ -1,25 +1,72 @@
 <div class="anim-fade-in-up">
-	<h2 class="section-header">
-		<translate>auth.join.almost.heading</translate>
-	</h2>
+	<template v-if="doingCaptcha">
+		<h2 class="section-header">
+			<translate>Are you a robot?</translate><br>
+			(ノಠ益ಠ)ノ彡┻━┻
+		</h2>
 
-	<div v-translate>
-		auth.join.almost.body_html
-	</div>
-	<p class="small text-muted">
-		<translate>auth.join.almost.spam</translate>
-	</p>
-	<p class="small text-muted">
-		<translate>auth.join.almost.urgency</translate>
-	</p>
+		<app-loading v-if="!recaptchaReady" />
+		<div v-else ref="recaptcha" key="recaptcha"/><br>
+		<div class="alert alert-warning" v-if="serverErrors">
+			<translate v-if="serverErrors.token">
+				Oh no, your sign-up request has expired!
+			</translate>
+			<translate v-else-if="serverErrors.email || serverErrors.username">
+				Uh oh, someone already registered with your email or username!
+			</translate>
+			<translate v-else-if="serverErrors.captcha">
+				Mind trying the captcha again?
+			</translate>
+			<translate v-else-if="serverErrors['rate-limit']">
+				Sorry, you can't register another account yet. Try again later.
+			</translate>
+			<translate v-else>
+				Uh oh, something weird happened!
+			</translate>
+			<br>
+			<button
+				class="btn"
+				@click="gotoJoin"
+				v-if="!serverErrors['rate-limit']"
+				>
+				<app-jolticon icon="reply" />
+				<translate>Retry?</translate>
+			</button>
+		</div>
+		<div v-translate>
```

Comment:
Change to <translate< wherever v-translate is not needed. We only need to use it when we have parameters or plural manipulations or inner html.